WORKSPACE

```diff
@@ -571,6 +571,14 @@ http_file(
     ],
 )
 
+http_file(
+    name = "virt-what",
+    sha256 = "fb940984d58924ea20fe1ac8e5c0aca9963331695cbc9942a3791c2c23ff1861",
+    urls = [
+        "https://dl.fedoraproject.org/pub/archive/fedora/linux/releases/28/Everything/x86_64/os/Packages/v/virt-what-1.18-5.fc28.x86_64.rpm",
```

Comment:
I'm just wondering why do we use fedora 28 package? I think we have already moved to fedora 32 guest image.

Reply:
Mostly I followed the dmidecode example. But Fedora doesn't have stable links for newer distro packages, they only become long term stable once the distro goes EOL (for reasons that have never been clear to me), so the oldest we could use with a stable long term link is Fedora 30. I will try running with that version and update the commit if it works

Reply:
f30 version works too so I updated the commit